client/my-sites/plans-features-main/hooks/data-store/use-pricing-meta-for-grid-plans.ts

```diff
@@ -119,13 +112,13 @@ const usePricingMetaForGridPlans: UsePricingMetaForGridPlans = ( {
 					 */
 					if ( purchasedPlan ) {
 						const isMonthly = purchasedPlan.billPeriodDays === PLAN_MONTHLY_PERIOD;
-						const isYearly = purchasedPlan.billPeriodDays === PLAN_ANNUAL_PERIOD;
 
 						if ( isMonthly && monthlyPrice !== purchasedPlan.priceInteger ) {
 							monthlyPrice = purchasedPlan.priceInteger;
-							yearlyPrice = calculateYearlyPrice( TERM_ANNUALLY, purchasedPlan.priceInteger );
-						} else if ( isYearly && yearlyPrice !== purchasedPlan.priceInteger ) {
-							monthlyPrice = calculateMonthlyPrice( TERM_ANNUALLY, purchasedPlan.priceInteger );
+							yearlyPrice = parseFloat( ( purchasedPlan.priceInteger * 12 ).toFixed( 2 ) );
+						} else if ( yearlyPrice !== purchasedPlan.priceInteger ) {
```

Comment:
Slightly confusing that the variable here is named `yearlyPrice`, since it's not necessarily a yearly plan.

Reply:
Good point. Replaced with `fullPrice`.